### PR TITLE
Adding styling sample for Grids ESF

### PR DESCRIPTION
**Pull request by rmkrmk**
IgniteUI/igniteui-docfx#1275

### CODE REVIEW

File: live-editing/configs/TreeGridConfigGenerator.ts [hunk 1]
```diff
@@ -387,6 +390,17 @@ export class TreeGridConfigGenerator implements IConfigGenerator {
             shortenComponentPathBy: "/tree-grid/"
         }));
 
+        configs.push(new Config({
+            additionalFiles: ["/src/app/tree-grid/data/foods.ts"],
+            appModuleConfig: new AppModuleConfig({
+                imports: [IgxTreeGridModule, TreeGridExcelStyleFilteringStyleComponent, IgxInputGroupModule],
```
Comment (tachojelev) [inline live-editing/configs/TreeGridConfigGenerator.ts]: Do we need the **IgxInputGroupModule** here?

File: live-editing/configs/TreeGridConfigGenerator.ts [hunk 2]
```diff
@@ -387,6 +390,17 @@ export class TreeGridConfigGenerator implements IConfigGenerator {
...
+                ngDeclarations: [TreeGridExcelStyleFilteringStyleComponent],
+                ngImports: [IgxTreeGridModule, IgxInputGroupModule]
```
Comment (tachojelev) [inline live-editing/configs/TreeGridConfigGenerator.ts]: Do we need the IgxInputGroupModule here?

File: src/app/hierarchical-grid/hierarchical-grid-excel-style-filtering-style/hierarchical-grid-excel-style-filtering-style.component.scss [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+
+.photo {
+    vertical-align: middle;
+    max-height: 62px;
+}
+.cell__inner_2 {
+    margin: 1px
+}
+
+.density-chooser {
```
Comment (tachojelev) [inline src/app/hierarchical-grid/hierarchical-grid-excel-style-filtering-style/hierarchical-grid-excel-style-filtering-style.component.scss]: This class does not seem to be used in the sample.

File: src/app/hierarchical-grid/hierarchical-grid-excel-style-filtering-style/hierarchical-grid-excel-style-filtering-style.component.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    public localdata;
+
+    constructor() {
+        this.localdata = SINGERS;
+    }
+
+    public ngOnInit(): void {
+    }
+
+    public formatter = (a) => a;
```
Comment (tachojelev) [inline src/app/hierarchical-grid/hierarchical-grid-excel-style-filtering-style/hierarchical-grid-excel-style-filtering-style.component.ts]: This formatted is not necessary, I believe we can remove it from sample.

File: live-editing/configs/TreeGridConfigGenerator.ts (hunk 1 above)
Comment (rmkrmk) [reply live-editing/configs/TreeGridConfigGenerator.ts]: Addressed.

File: live-editing/configs/TreeGridConfigGenerator.ts (hunk 2 above)
Comment (rmkrmk) [reply live-editing/configs/TreeGridConfigGenerator.ts]: Addressed.

File: src/app/hierarchical-grid/hierarchical-grid-excel-style-filtering-style/hierarchical-grid-excel-style-filtering-style.component.scss (hunk 3 above)
Comment (rmkrmk) [reply src/app/hierarchical-grid/hierarchical-grid-excel-style-filtering-style/hierarchical-grid-excel-style-filtering-style.component.scss]: Addressed.

File: src/app/hierarchical-grid/hierarchical-grid-excel-style-filtering-style/hierarchical-grid-excel-style-filtering-style.component.ts (hunk 4 above)
Comment (rmkrmk) [reply src/app/hierarchical-grid/hierarchical-grid-excel-style-filtering-style/hierarchical-grid-excel-style-filtering-style.component.ts]: Addressed.

**Comment by astaev** [comment]
Some buttons at the second filter dialog should be styled as well:
ESPdialog

**Comment by rmkrmk** [comment]
ESF sub-dialogs styling addressed.

**Pull request merged**